MilitaryManager.Units.API/appsettings.Development.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "Data Source=BAGUTA;Initial Catalog=Units;User ID=sa;Password=password1;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
+    "DefaultConnection": "Data Source=localhost,1433;Initial Catalog=Units;User ID=User_2;Password=qwe123qwe;Connect Timeout=30;Encrypt=False;TrustServerCertificate=False;ApplicationIntent=ReadWrite;MultiSubnetFailover=False"
```

Comment:
Please, don't use user and password for local connection, it's just enough for Windows Authentication (integrated security)